modules/FindAssimp.cmake

```diff
@@ -0,0 +1,92 @@
+#.rst:
+# Find Assimp
+# -------------
+#
+# Finds the Assimp library. This module defines:
+#
+#  Assimp_FOUND           - True if Assimp library is found
+#  Assimp::Assimp         - Assimp imported target
+#
+# Additionally these variables are defined for internal usage:
+#
+#  ASSIMP_LIBRARY         - Assimp library
+#  ASSIMP_LIBRARIES       - Same as ASSIMP_LIBRARY
+#  ASSIMP_INCLUDE_DIR     - Include dir
+#
+
+#
+#   This file is part of Magnum.
+#
+#   Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016
+#             Vladimír Vondruš <mosra@centrum.cz>
```

Comment:
Could you put 2017 in all the copyright headers so it's in sync with all the other code? Thank you :)